exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -205,10 +225,10 @@ drill.exec: {
         // Deprecated for managed xsort; used only by legacy xsort
         threshold: 40000,
         // File system to use. Local file system by default.
-        fs: "file:///"
+        fs: ${drill.exec.spill.fs},
```

Comment:
Done. Added:

        // -- The two options below can be used to override the options common
        // -- for all spilling operators (see "spill" above).
        // -- This is done for backward compatibility; in the future they
        // -- would be deprecated (you should be using only the common ones)